external-deps/spyder-kernels/.gitrepo

```diff
@@ -5,8 +5,8 @@
 ;
 [subrepo]
 	remote = https://github.com/spyder-ide/spyder-kernels.git
-	branch = 2.x
-	commit = ac40350c9260e2e5d73f0fc3f8a9cc15da8d8a03
-	parent = 01421c2ae26c7a3861fc4c912f295f9f2b2c1c1f
+	branch = client7
```

Comment:
I mean, you probably don't want to merge while this is referenced.

Reply:
Right, I'll merge your corresponding PR in spyder-kernels, then ask you to resync our subrepo here, then merge.

The process is explained in our [contributing guide](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md).